document CO `[ChannelN],input_configured` (#655)
mixxxdj/manual: source/chapters/appendix/mixxx_controls.rst

.. note:: Although the first auxiliary group is named :mixxx:cogroupref:`[Auxiliary1]`, the group for the first microphone is just called :mixxx:cogroupref:`[Microphone] <[MicrophoneN]>`, not :mixxx:cogroupref:`[Microphone1] <[MicrophoneN]>`.

.. mixxx:control:: [ChannelN],input_configured
                   [MicrophoneN],input_configured
                   [AuxiliaryN],input_configured

   1 if there is input is configured for this channel, 0 if not.
   In the case of :mixxx:cogroupref:`[ChannelN]` it corresponds to

>    Vinyl Control. This should be checked before enabling :mixxx:coref:`[ChannelN],passthrough`
>    as you can't passthrough an input channel that is not configured.

Reviewer: ```suggestion
   Vinyl Control. A configured input is required to enable :mixxx:coref:`[ChannelN],passthrough`
```
?
